dockstore-client/src/main/java/io/dockstore/client/cli/nested/AbstractEntryClient.java

```diff
@@ -1249,9 +1265,72 @@ public void loadDockerImages() {
 
     public abstract Client getClient();
 
+
+
     /**
      * help text output
      */
+    public void wesHelp() {
+        printHelpHeader();
+        out("Commands:");
+        out("");
+        out("Usage: dockstore " + getEntryType().toLowerCase() + " wes --help");
+        out("       dockstore " + getEntryType().toLowerCase() + " wes launch [parameters]");
+        out("       dockstore " + getEntryType().toLowerCase() + " wes status [parameters]");
+        out("       dockstore " + getEntryType().toLowerCase() + " wes cancel [parameters]");
+        out("");
+        out("Description:");
+        out(" Executes a command on a Workflow Execution Service (WES) endpoint.");
+        printWesHelpFooter();
+        printHelpFooter();
+    }
+
+    protected void wesLaunchHelp() {
+        printHelpHeader();
+        out("Usage: dockstore wes " + getEntryType().toLowerCase() + " launch --help");
+        out("       dockstore wes " + getEntryType().toLowerCase() + " launch [parameters]");
+        printLaunchHelpBody();
+        printWesHelpFooter();
+        printHelpFooter();
+    }
+
+    public void wesStatusHelp() {
+        printHelpHeader();
+        out("Usage: dockstore " + getEntryType().toLowerCase() + " wes status --help");
+        out("       dockstore " + getEntryType().toLowerCase() + " wes status [parameters]");
+        out("");
+        out("Description:");
+        out("  Status, gets the status of a " + getEntryType() + ".");
+        out("Required Parameters:");
+        out("  --id <id>                           Id of a " + getEntryType() + " at the WES endpoint, e.g. id returned from the launch command");
```

Comment:
Isn't this the id of the WES job? It's not the id of the workflow or a tool is it? This message implies to me that it's the id of the workflow or tool.

More generally, and beyond the scope of this ticket, I wonder if the status and other operations should be even be part of workflow/tool command. IOW, I think I can do this (I may not have the exact syntax)

1. dockstore **workflow** wes launch....
1. Save the id from step 1
1. dockstore **tool** wes status <id from above>

This will work, won't it? There's no enforcement, nor could there be, of the type of tool/workflow that was launched.

If I'm wrong, ignore this whole comment. If I'm right, I suggest creating another ticket to reconsider the command line options and tweaking this message to say it's the job id.

Or maybe I'm being too nitpicky.

Reply:
Changed wording from 'Id of a workflow' to 'Id of a run'. Right now 'tool' is not permitted.